docs-src/cloud/how-to-monitor-export-progress.md

```diff
@@ -0,0 +1,47 @@
+---
+id: how-to-monitor-export-progress
+title: Monitor export progress
+sidebar_label: Monitor
+description: Track of the progress of your Workflow History Exports in Temporal Cloud.
+tags:
+  - temporal cloud
+  - workflow history export
+  - how-to
+---
+
+**How to monitor the History export progress**
+
+Once you've finalized the setup, here's how to monitor the export progress:
+
+1. **Export Job Execution**:
+   - **Schedule**: The Export job is scheduled to run on an hourly basis, starting at 10 minutes past each hour.
+     This ensures that the history data of Closed Workflows is exported to your designated S3 bucket approximately 60 minutes post Workflow closure. This delay can be configured.
```

Comment:
Suggest we remove this line as under delivery guarantee we talked about the delay more specifically.